### PR TITLE
Update CHANGELOG.md for 1.6.0

### PR DESCRIPTION
**Pull request by oliverchang**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,37 @@
...
+- [Bug #703](https://github.com/google/osv-scanner/pull/703)
+  Return an error if both license scanning and local/offline scanning is enabled simultaneously.
+
+- [Bug #718](https://github.com/google/osv-scanner/pull/718)
+  Fixed parsing of SBOMs generated by the latest CycloneDX.
+
+- [Bug #704](https://github.com/google/osv-scanner/pull/704)
+  Get go stdlib version from go.mod.
+
+### Miscellaneous
```
Comment (another-rex) [inline CHANGELOG.md:34]: I think I have these with `API Features` heading, and we can add #727 as the PR number.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,37 @@
...
+
+### Features
+- [Feature #694](https://github.com/google/osv-scanner/pull/694)
+  Add support for NuGet lock files version 2 by @giovanni-bozzano
+
+- [Feature #655](https://github.com/google/osv-scanner/pull/655)
+  Scan and report dependency groups (e.g. "dev dependencies") for vulnerabilities
+
+- [Feature #702](https://github.com/google/osv-scanner/pull/702)
+  Created an option to skip/disable upload to code scanning by @another-rex in
```
Comment (another-rex) [inline CHANGELOG.md]: Unfinished sentence?

**Review by another-rex — COMMENTED** [review]
I generally don't add the credit in the changelog, as people can just click into the PR to see who added it. 

In the actual release a section for first time contributors is also generated by github.

File: CHANGELOG.md (hunk 1 above)
Comment (oliverchang) [reply CHANGELOG.md]: done!

File: CHANGELOG.md (hunk 2 above)
Comment (oliverchang) [reply CHANGELOG.md]: stripped the credits properly this time.

**Review by another-rex — APPROVED** [review]

**Pull request merged**